commit 76b40406aa8aed820df8f1384f1aa8e62d1108c3
Author: Mike Bayer <mike_mp@zzzcomputing.com>
Date:   Tue Mar 3 14:32:56 2020 -0500

    Remove oslo_db.sqlalchemy.compat reference
    
    oslo.db has removed this module in
    I63e4baf772be9ddfb787ac3aff01fcaddf7b901c.  To the degree that
    it is used in this Nova module is for a single check of SQLAlchemy
    prior to 0.9.0 which is well below the minimum SQLAlchemy version
    of 1.2.19.
    
    Change-Id: I062da797ae4a4961535155a19ad330f5e8e14950

diff --git a/nova/tests/unit/db/test_migration_utils.py b/nova/tests/unit/db/test_migration_utils.py
index a599294a75..a4678fe7ea 100644
--- a/nova/tests/unit/db/test_migration_utils.py
+++ b/nova/tests/unit/db/test_migration_utils.py
@@ -13,7 +13,6 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from oslo_db.sqlalchemy.compat import utils as compat_utils
 from oslo_db.sqlalchemy import enginefacade
 from oslo_db.sqlalchemy import test_base
 from oslo_db.sqlalchemy import test_fixtures
@@ -31,8 +30,6 @@ from nova import exception
 from nova import test
 from nova.tests import fixtures as nova_fixtures
 
-SA_VERSION = compat_utils.SQLA_VERSION
-
 
 class CustomType(UserDefinedType):
     """Dummy column type for testing unsupported types."""
@@ -188,12 +185,6 @@ class TestMigrationUtilsSQLite(
                       Column('a', CustomType))
         table.create()
 
-        # reflection of custom types has been fixed upstream
-        if SA_VERSION < (0, 9, 0):
-            self.assertRaises(oslodbutils.ColumnError,
-                              utils.create_shadow_table,
-                              self.engine, table_name=table_name)
-
         utils.create_shadow_table(self.engine,
                                   table_name=table_name,
                                   a=Column('a', CustomType()))
